src/io.F

```diff
@@ -1,867 +1,870 @@
-C----------------------------------------------------------------------
```

Comment:
IO changes still here. If truly necessary they should be in a separate pull request explaining why these changes are being made.